src/i18n/translations/en/translation.json

```diff
@@ -47,13 +47,13 @@
     "popularDocs": {
       "title": "Popular docs",
       "t1": {
-        "title": "NRQL syntax, clauses, and functions"
+        "title": "Monitor all your apps and microservices"
       },
       "t2": {
-        "title": "Forward logs using infrastructure agent"
+        "title": "Overcome security blindspots with Vulnerability Management"
       },
       "t3": {
-        "title": "Install the Java agent"
+        "title": "Set up New Relic alerts with Terraform"
```

Comment:
Seems a bit unsure when we're launching Terraform because of the missing component. I'd just cut this for now and we can open a second PR in a few days!